demos/chips.html

```diff
@@ -54,18 +54,82 @@
       </section>
 
       <section class="example">
+        <h2>Entry Chips</h2>
         <div class="mdc-chip-set">
-          <div class="mdc-chip">
-            <div class="mdc-chip__text">Add to Calendar</div>
+          <div class="demo-chip mdc-chip mdc-chip--with-leading-icon mdc-chip--with-trailing-icon">
+            <i class="material-icons mdc-chip__icon" tabindex="0">face</i>
```

Comment:
If we're going to have `tabindex="0"` here, we should also add `:focus` and `:hover` styles for leading icons.

Reply:
Removed tabindex since interacting with the leading icon should be the same as interacting with the chip.